src/Header/Nav/Nav.jsx

```diff
@@ -61,7 +61,9 @@ function Nav({
 	// See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find for support
 	let activeService = null;
 	let internalService = false;
-	let servicesToDisplay = services.external; //default to displaying external services.
+	let servicesToDisplay = services.external.filter(
+		(service) => service.header === true
+	); // only display external services with header=true
```

Comment:
if you're filtering here, can the same filter for that prop in navlinks module be removed?

https://github.com/nice-digital/global-nav/blob/c22223f082a48dfa3bb0a1c2232ac724a4079258/src/Header/Nav/NavLinks/NavLinks.jsx#L77